.circleci/config.yml

```diff
@@ -499,52 +524,62 @@ workflows:
   version: 2.1
   commit:
     jobs:
-      - run-tests:
-          # Need to tell job run-tests to been launched for all tags
-          # as it is a required step for some jobs, triggered by tags
-          # see: https://discuss.circleci.com/t/tag-filter-not-firing-in-workflow/25534/3
+      - checkout-and-install-deps:
           filters:
             tags:
               only: /.*/
+      - test-lint-and-types:
+          requires:
+            - checkout-and-install-deps
+      - test-native:
+          requires:
+            - checkout-and-install-deps
+      - test-web:
+          requires:
+            - checkout-and-install-deps
       - deploy-soft-testing:
           filters:
             branches:
               only:
                 - master
           requires:
-            - run-tests
+            - checkout-and-install-deps
       - build-android-production-hard:
           filters:
             tags:
               ignore: /.*/
             branches:
               only: master
           requires:
-            - run-tests
+            - checkout-and-install-deps
```

Comment:
besoin de `checkout-and-install-deps` ou il est déjà demandé par `test-lint-and-types` ?